config/replacements.php

```diff
@@ -3,6 +3,7 @@
 return [
     // NEVER REWRITE
     'zendframework/zendframework' => 'zendframework/zendframework',
+    'Doctrine\\Zend' => 'Doctrine\\Zend',
```

Comment:
Shouldn't we also include the `Doctrine\\\\Zend` variant?

Reply:
I was thinking about it, and as there is no module of doctrine-zend-hydrator package I think we don't need it.